test/unit/html_attachment_test.rb

```diff
@@ -48,6 +48,7 @@ class HtmlAttachmentTest < ActiveSupport::TestCase
     edition = create(:draft_publication, :with_html_attachment)
     attachment = edition.attachments.first
     attachment.update!(locale: "fi")
+    Timecop.travel(11.years) { edition.update!(updated_at: Time.zone.now) }
```

Comment:
The `11.years` is an arbitrary choice, repeated throughout - could you extract it to a variable somewhere and then just refer to that?

```ruby
let(:to_the_future) { 11.years }

Timecop.travel(to_the_future) { ...
```

Reply:
I probably could explicitly set this to be after 29 November 2021 to be honest — the 11 years was just based on the date at which the tests were already frozen at. But also, these tests would be expected to be removed once the migration goes ahead.